rllib/agents/ppo/ddppo.py

```diff
@@ -21,14 +21,16 @@
 import time
 
 import ray
-from ray.rllib.agents.ppo import ppo
+from ray.rllib.agents.ppo.ppo import DEFAULT_CONFIG as PPO_DEFAULT_CONFIG, \
+    PPOTrainer
```

Comment:
this doesn't fit in the last line?

Reply:
Not sure what you mean. The line is too long for the LINTer and needs to be split by a `\`

Reply:
ah sorry, it looked really strange on my laptop. my bad.